user-org-util/src/main/java/org/sunbird/util/jsonkey/UserJsonKey.java

```diff
@@ -5,6 +5,7 @@
  */
 public interface UserJsonKey {
 
+    String OSID = "osid";
```

Comment:
move this constant under Jsonkey file as it will be used across the application not only user related  file.

Reply:
This must not be hardcoded. Remember we have an env var uuidPropertyName.

Reply:
Yes it is configurable property. Now got this value from DbConnectionInfoMngr bean.